examples/qualcomm/qaihub_scripts/utils/utils.py

```diff
@@ -68,7 +68,6 @@ def gen_pte_from_ctx_bin(
     for pte_name in pte_names:
         print(f"{pte_name} generating...")
         memory_planning_pass = MemoryPlanningPass(
-            memory_planning_algo="greedy",
```

Comment:
[The default algo is greedy](https://github.com/pytorch/executorch/blob/e12b37e83885342b7664f7434a3988ab5c6cc111/exir/passes/memory_planning_pass.py#L30) and I'm not sure how this line will fix the issue. Could you elaborate a bit more?